Sources/BraintreeDropIn/Models/BTDropInResult.m

```diff
@@ -133,10 +133,15 @@ + (void)mostRecentPaymentMethodForAPIClient:(BTAPIClient * _Nullable)apiClient
                 BTPaymentMethodNonce *paymentMethod = paymentMethodNonces.firstObject;
                 result.paymentMethodType = [BTUIKViewUtil paymentMethodTypeForPaymentInfoType:paymentMethod.type];
                 result.paymentMethod = paymentMethod;
+                completion(result, nil);
+                return;
+            } else {
```

Comment:
nit: I wonder if this would be more readable if we flipped this logic and did the error first if the payment method count is equal to 0. Not a blocker at all though!

Reply:
Great feedback, will update!

Reply:
Pushed!